apps/linux/firefox.talon

```diff
@@ -72,3 +72,5 @@ action(browser.submit_form):
 
 action(browser.toggle_dev_tools):
 	key(ctrl-shift-i)
+
+select all: key(ctrl-a)
```

Comment:
I can't believe I missed this command. There's an existing action that's implemented in edit.talon, `edit.select_all`

I think 'select all' should be moved here - 
https://github.com/knausj85/knausj_talon/blob/master/text/generic_editor.talon

Reply:
sounds good, moved it and just pushed it for this PR